controllers/alphacontrollers/alert_controller.go

```diff
@@ -992,8 +995,17 @@ func flattenNotification(notification *alerts.AlertNotification) coralogixv1alph
 
 	switch integration := notification.GetIntegrationType().(type) {
 	case *alerts.AlertNotification_IntegrationId:
-		flattenedNotification.IntegrationID = new(int32)
-		*flattenedNotification.IntegrationID = int32(integration.IntegrationId.GetValue())
+		id := strconv.Itoa(int(integration.IntegrationId.GetValue()))
+		webhookStr, err := coralogixv1alpha1.WebhooksClient.GetWebhook(context.Background(), id)
+		if err != nil {
```

Comment:
not sure how we want to handle APIs errors

Reply:
If we don't have think kind of knowledge, I guess it is fine to fail the reconciliation

Reply:
I think it would be better to leave it to reconciliation anyway to avoid complicate logic here

Reply:
Yeah, let's just return the error and decide what to do with it during reconciliation